packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -951,55 +951,62 @@ class _InteractiveViewerState extends State<InteractiveViewer> with TickerProvid
 
   // Handle mousewheel scroll events.
   void _receivedPointerSignal(PointerSignalEvent event) {
+    double scaleChange = 1;
```

Comment:
Are you still able to use `final` here?  I always forget when that works or not...

Or `late final` if not.